src/Tribe/Deprecation.php

```diff
@@ -30,6 +30,7 @@ class Tribe__Deprecation {
 	protected $deprecated_filters = array(
 		'tribe_cost_regex'            => array( '4.3', 'tribe_events_cost_regex' ),
 		'tribe_rewrite_prepared_slug' => array( '4.3', 'tribe_events_rewrite_prepared_slug' ),
+		''                            => array( 'TBD', 'tribe_events_linked_post_type_meta_key' ),
```

Comment:
Why empty key? That doesn't seem right.

Reply:
@bordoni this property's docblock states:
> Use the format `<new_filter_tag> => ...`

There is no new filter. What should I do instead?

Reply:
Interesting one! I can think of two possibilities here. First, and requiring the least work, we could use a convention like `No replacement {tag}`:

- Give us a unique key
- Will result in safe (and useful-when-debugging) information being returned by _Tribe__Deprecation::get_filter_for_deprecated_tag()_

Alternatively, requiring a _little_ more work, if there is no replacement we could fall back on numerical indexing, ie:

```php
array(
 		'tribe_cost_regex'            => array( '4.3', 'tribe_events_cost_regex' ),
 		'tribe_rewrite_prepared_slug' => array( '4.3', 'tribe_events_rewrite_prepared_slug' ),
		array( 'TBD', 'tribe_events_linked_post_type_meta_key' ), # No replacement
);
```

If we did this, we'd also want to update our _Tribe__Deprecation::get_FILTER|ACTION_for_deprecated_tag()_ methods to detect that the key is numerical. They can then dynamically compose a return value like `'No alternative for $tag`.